src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>Then again, we have the same result as what we found in <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/77d98d3c37a399077ed2096421f05fdebf4b6230/src/fastoad/notebooks/02_explanation/02_pure_python.ipynb" target="_blank">the second notebook</a>. </blockquote><p> </p><p>I suggest:</p><p> </p><p>Then again, our result is very close to what we found in <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/77d98d3c37a399077ed2096421f05fdebf4b6230/src/fastoad/notebooks/02_explanation/02_pure_python.ipynb" target="_blank">the second notebook</a> (slightly better, actually). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Changed in 7a6643b4</p>